torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -239,6 +239,15 @@ RegisterOperators reg(
              return 0;
            };
          }),
+     Operator(
+         "prim::GraphHolder() -> ()",
+         [](const Node* /* node */) {
+           return [](Stack& stack) {
+             // Returns an int so that we have an easy way to do graph traversal
```

Comment:
This should throw.

Reply:
Going to leave this as a follow up so that I don't muck around with the bottom of this stack too much. Added a TODO comment here in the code